x-pack/plugins/apm/public/components/app/service_overview/service_overview.test.tsx

```diff
@@ -86,6 +86,8 @@ describe('ServiceOverview', () => {
         isAggregationAccurate: true,
       },
       'GET /api/apm/services/{serviceName}/dependencies': [],
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      'GET /api/apm/services/{serviceName}/service_overview_instances': [],
```

Comment:
What is eslint complaining about here?

Reply:
`Property name `GET /api/apm/services/{serviceName}/error_groups` must match one of the following formats: camelCase, PascalCase, snake_case, UPPER_CASE`